src/shared/is_node.js

```diff
@@ -16,11 +16,12 @@
 
 // NW.js / Electron is a browser context, but copies some Node.js objects; see
 // http://docs.nwjs.io/en/latest/For%20Users/Advanced/JavaScript%20Contexts%20in%20NW.js/#access-nodejs-and-nwjs-api-in-browser-context
-// https://electronjs.org/docs/api/process#processversionselectron
+// https://www.electronjs.org/docs/api/process#processversionselectron
```

Comment:
If you're touching this line anyway, please also fix the link to point directly to the relevant section: https://www.electronjs.org/docs/api/process#processversionselectron-readonly

Reply:
I have edited as requested.